examples/cron-backfill.yaml

```diff
@@ -35,6 +35,7 @@ spec:
   # run daily at 2am
   schedule: "0 2 * * *"
   workflowSpec:
+    entrypoint: main
```

Comment:
why don't we use the `entrypoint` from the template? this feels like a bug to me

Reply:
`WorkflowTemplate` job doesn't have an Entrypoint.  We can add it in `WorkflowTemplate` or workflow.

Reply:
can this be in the template please?